src/lib.rs

```diff
@@ -2074,7 +2109,7 @@ impl Connection {
     /// Continues the handshake.
     ///
     /// If the connection is already established, it does nothing.
```

Comment:
Maybe we should describe the parameter `now`?

Reply:
IDK, to me it seems fairly self-explanatory what a `std::time::Instant` variable called `now` represents TBH.